frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -92,6 +92,36 @@ export const BucketsItem: React.FC<BucketsItemProps> = React.memo(
   ),
 );
 
+export const BucketsTitle: React.FC<BucketsTitleProps> = ({
+  objectsCount,
+  link,
+  error,
+  children,
+}) => {
+  let objectsBody;
+  if (!objectsCount && !error) {
```

Comment:
Are we planning to show loading if objects count is 0?

Reply:
No